src/js/components/LanguageWrapper.js

```diff
@@ -110,10 +110,12 @@ class LanguageWrapper extends Component {
   render() {
     const { lang } = this.state;
     const messages = localeMessages[lang];
+    // Right to Left Direction for arabic text
+    const direction = lang === 'ar' ? 'rtl' : 'ltr';
 
     return (
       <IntlProvider locale={lang} messages={messages} defaultLocale={DEFAULT_LANG} key={lang}>
-        <div style={{ position: 'relative' }}>
+        <div style={{ position: 'relative' }} dir={direction} className={direction}>
```

Comment:
This is the important part. If our lang is Arabic, throw on the `rtl` class name and `dir` value for everything in the inheritance below. `dir` takes care of a lot of things automatically in the browser. For everything else, we namespace some CSS rules under `.rtl { ... }`